Dockerfile

```diff
@@ -47,6 +47,8 @@ RUN mvn clean install -T1C -U -f /src/dhis-2/dhis-web/pom.xml
 FROM tomcat:8.5.34-jre8-alpine as serve
 
 RUN rm -rf /usr/local/tomcat/webapps/*
+
+COPY server.xml /usr/local/tomcat/conf
```

Comment:
This does a bunch of stuff, including several good things (like enabling [ ] { } characters in a querystring!) is there anything it does that we wouldn't want in some situations?

Reply:
This is default server.xml i copied from tomcat started by docker. Only updated query string args section